discovery-provider/src/monitors/filesystem.py

```diff
@@ -0,0 +1,11 @@
+import psutil
+
+
+def get_filesystem_size(**kwargs):
+    disk = psutil.disk_usage('/')
+    return disk.total
+
+
+def get_filesystem_used(**kwargs):
```

Comment:
why do these functions need **kwargs?

Reply:
i see why, it appears that we always pass in db/redis and only conditionally use them - i think a comment explaining that would be useful

Reply:
added!